client/my-sites/all-sites/docs/example.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import AllSites from 'my-sites/all-sites';
+import Card from 'components/card';
+
+const AllSitesExample = () => (
```

Comment:
The issue is for the wpcalypso, since we mangle variable names in prod environments. https://wpcalypso.wordpress.com/devdocs/blocks/u

![screen shot 2017-05-25 at 9 41 10 am](https://cloud.githubusercontent.com/assets/1270189/26460390/5938b844-412e-11e7-846e-959eab4783e0.png)